<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordsLearning</class>
 <widget class="QWidget" name="WordsLearning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>单词本</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>250</width>
                <height>37</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="selectLabel">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>42</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑 Light</family>
                <pointsize>9</pointsize>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url();
font: 25 9pt '微软雅黑 Light';
color: rgb(31,31,31);
padding-left:20px;
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);border-radius:15px;</string>
              </property>
              <property name="text">
               <string>请选择你要学习的单词本    </string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>249</width>
                <height>37</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="CET4WordsBtn">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>220</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {  
    background-color: transparent;  
    border: none;  
    border-radius: 10px; /* 设置圆角半径 */  
}  
  
QPushButton:hover {  
    background-color: #ADD8E6; /* 浅蓝色 */  
    border-radius: 10px; /* 保持圆角半径，确保悬停时形状不变 */  
}</string>
              </property>
              <property name="text">
               <string>四级</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="CET6WordsBtn">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>220</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {  
    background-color: transparent;  
    border: none;  
    border-radius: 10px; /* 设置圆角半径 */  
}  
  
QPushButton:hover {  
    background-color: #ADD8E6; /* 浅蓝色 */  
    border-radius: 10px; /* 保持圆角半径，确保悬停时形状不变 */  
}</string>
              </property>
              <property name="text">
               <string>六级</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="rememberWordsBtn">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>220</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {  
    background-color: transparent;  
    border: none;  
    border-radius: 10px; /* 设置圆角半径 */  
}  
  
QPushButton:hover {  
    background-color: #ADD8E6; /* 浅蓝色 */  
    border-radius: 10px; /* 保持圆角半径，确保悬停时形状不变 */  
}</string>
              </property>
              <property name="text">
               <string>熟词</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="toLearnBtn">
              <property name="minimumSize">
               <size>
                <width>161</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>点击开始学习单词</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>推荐学习单词</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QTableView" name="tableView">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="cursor" stdset="0">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="sortingEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="widget" native="true">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <widget class="QPushButton" name="learnWordBtn">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>210</y>
                <width>131</width>
                <height>31</height>
               </rect>
              </property>
              <property name="cursor">
               <cursorShape>OpenHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url();
font: 25 11pt '微软雅黑 Light';
color: rgb(31,31,31);
padding-left:20px;
background-color: rgb(255, 255, 255);
border:2px solid rgb(0,0,0);border-radius:15px;</string>
              </property>
              <property name="text">
               <string>学习单词</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="searchLineEdit">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>270</y>
                <width>131</width>
                <height>30</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url();
font: 25 11pt '微软雅黑 Light';
color: rgb(31,31,31);
padding-left:20px;
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);border-radius:15px;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignJustify|Qt::AlignVCenter</set>
              </property>
              <property name="placeholderText">
               <string>请输入单词：</string>
              </property>
             </widget>
             <widget class="QPushButton" name="searchWordBtn">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>310</y>
                <width>131</width>
                <height>31</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url();
font: 25 11pt '微软雅黑 Light';
color: rgb(31,31,31);
padding-left:20px;
background-color: rgb(255, 255, 255);
border:2px solid rgb(0,0,0);border-radius:15px;</string>
              </property>
              <property name="text">
               <string>搜索单词</string>
              </property>
             </widget>
             <widget class="QPushButton" name="returnWordsBtn">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>380</y>
                <width>131</width>
                <height>31</height>
               </rect>
              </property>
              <property name="cursor">
               <cursorShape>ClosedHandCursor</cursorShape>
              </property>
              <property name="toolTip">
               <string/>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url();
font: 25 11pt '微软雅黑 Light';
color: rgb(31,31,31);
padding-left:20px;
background-color: rgb(255, 255, 255);
border:2px solid rgb(0,0,0);border-radius:15px;</string>
              </property>
              <property name="text">
               <string>返回单词本</string>
              </property>
             </widget>
             <widget class="QLabel" name="learnedWoldLabel">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>170</y>
                <width>131</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑 Light</family>
                <pointsize>9</pointsize>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url();
font: 25 9pt '微软雅黑 Light';
color: rgb(31,31,31);
padding-left:20px;
background-color: rgb(255, 255, 255);
border:2px solid rgb(20,196,188);border-radius:15px;</string>
              </property>
              <property name="text">
               <string>已学习单词：</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignJustify|Qt::AlignVCenter</set>
              </property>
             </widget>
             <widget class="QLabel" name="wordIMGLabel">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>30</y>
                <width>111</width>
                <height>111</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
